### PR TITLE
Update config.cfg

### PR DESCRIPTION
**Pull request by povohat**
added updated qtv api url to fix json error in console on launch

### CODE REVIEW

File: ezquake/configs/config.cfg [hunk 1]
```diff
@@ -195,7 +198,7 @@ hud_teamfrags_place                   "gameclock"
 hud_teamfrags_rows                    "4"
 hud_teamfrags_show                    "0"
 hud_teamfrags_vertical                "1"
-hud_tracking_format                   "Tracking: %n �%t�"
```
Comment (drzel) [inline ezquake/configs/config.cfg:198]: Any idea what happened here?

Comment (povohat) [reply ezquake/configs/config.cfg:198]: nope! for some reason it thinks i removed then added an identical line?

Comment (drzel) [reply ezquake/configs/config.cfg:198]: Bizarre, well as long as there's no weird blank character in there that will break things - did you test with this cfg?

Comment (povohat) [reply ezquake/configs/config.cfg:198]: Did a quick run around and some demo playback as the line relates to spectating or viewing MVDs. The HUD element appeared as normal and obviously the JSON error on launch is gone.

**Pull request merged**